hedera-mirror-rest/topicmessage.js

```diff
@@ -21,6 +21,9 @@ import {InvalidArgumentError, NotFoundError} from './errors';
 import {TopicMessage} from './model';
 import * as utils from './utils';
 import {TopicMessageViewModel} from './viewmodel';
+import _ from 'lodash';
+import config from './config.js';
+import TopicMessageLookup from './model/topicMessageLookup.js';
```

Comment:
```suggestion
import config from './config';
import {TopicMessageLookup} from './model';
```

You'll need to import and export `TopicMessageLookup` model in `model/index.js`